src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReferenceHadoopSourceUnitTest.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.coders.VoidCoder;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.cloud.dataflow.sdk.transforms.Create;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.transforms.View;
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import com.google.cloud.dataflow.sdk.values.PCollectionView;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReferenceFileSource;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReferenceHadoopSource;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Map;
+
+public class ReferenceHadoopSourceUnitTest extends BaseTest implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Test
+    public void testReferenceSourceQuery() throws IOException {
+        final String referencePath = hg19MiniReference;
+
+        final ReferenceHadoopSource refSource = new ReferenceHadoopSource(referencePath);
+        final ReferenceBases bases = refSource.getReferenceBases(PipelineOptionsFactory.create(),
```

Comment:
Please add a comment explaining where hdfs comes in here. Looking at this, it looks like it's just reading a local file.